Source/AssetRipper.Processing/AnimationClips/AnimationClipConverter.cs

```diff
@@ -167,38 +180,35 @@ private void ProcessDenses(IClip clip, IAnimationClipBindingConstant bindings)
 					}
 					else if (binding.CustomType == (byte)BindingCustomType.None)
 					{
-						AddDefaultCurve(binding, path, time, dense.SampleArray[framePosition]);
+						AddDefaultCurve(binding, path, time, dense.SampleArray[framePosition], 0, 0);
 						curveIndex++;
 					}
 					else
 					{
-						AddCustomCurve(bindings, binding, path, time, dense.SampleArray[framePosition]);
+						AddCustomCurve(binding, path, time, dense.SampleArray[framePosition], 0, 0);
 						curveIndex++;
 					}
 				}
 			}
 			ArrayPool<float>.Shared.Return(rentedArray);
 		}
 
-		private void ProcessConstant(IClip clip, IConstantClip constant, IAnimationClipBindingConstant bindings, float lastFrame)
+		private void ProcessConstant(IConstantClip constant, int preConstantCurves, float lastFrame)
 		{
-			float[] rentedArray = ArrayPool<float>.Shared.Rent(constant.Data.Count);
-			constant.Data.CopyTo(rentedArray);
-			ReadOnlySpan<float> curveValues = new ReadOnlySpan<float>(rentedArray, 0, constant.Data.Count);
-
 			ReadOnlySpan<float> slopeValues = [0, 0, 0, 0]; // no slopes - 0 values
 
-			int streamCount = clip.StreamedClip.CurveCount();
-			int denseCount = (int)clip.DenseClip.CurveCount;
+			float[] rentedArray = ArrayPool<float>.Shared.Rent(constant.Data.Count);
+			constant.Data.CopyTo(rentedArray);
+			ReadOnlySpan<float> curveValues = new(rentedArray, 0, constant.Data.Count);
```

Comment:
Please move these back. It clutters the diff and makes the pull request take longer to review.